README.md

```diff
@@ -176,6 +207,63 @@ GTO the artifact file is committed to Git.
 
 </details>
 
+### Deprecating a version
```

Comment:
Can we call it `unregister`/`unpublish` a version? So that it is clear that this is the opposite of `register`/`publish` a version? Just like we have `assign` and `unassign`.

Reply:
Good idea! Yes, we should do that I think, if we're ok with the names.

Reply:
resolving thread since we decided to implement `tag`/`untag` for now